Packages/packages-lock.json

```diff
@@ -137,8 +137,7 @@
       "depth": 0,
       "source": "builtin",
       "dependencies": {
-        "com.unity.modules.ui": "1.0.0",
-        "com.unity.modules.imgui": "1.0.0"
+        "com.unity.modules.ui": "1.0.0"
```

Comment:
I wonder why this keeps changing over and over again, whenever someone makes a pull request. Do you have some ideas?

Reply:
Imo using one version of Unity will be the solution.